pkg/network/config/config.go

```diff
@@ -241,6 +241,9 @@ type Config struct {
 	// for things like creating netlink sockets for conntrack updates, etc.
 	EnableRootNetNs bool
 
+	// HTTP2DynamicTableMapCleanerInterval is the interval to run the cleaner function.
+	HTTP2DynamicTableMapCleanerInterval time.Duration
```

Comment:
How is this different from `HTTPMapCleanerInterval`?

Reply:
30s by default verses 300s